cmd/s3-authmate/main.go

```diff
@@ -440,9 +440,16 @@ func parsePolicies(val string) (authmate.ContainerPolicies, error) {
 	if val == "" {
 		return nil, nil
 	}
-	data, err := os.ReadFile(val)
-	if err != nil {
+
+	var (
 		data = []byte(val)
+		err  error
+	)
+
+	if !json.Valid(data) {
+		if data, err = os.ReadFile(val); err != nil {
+			return nil, fmt.Errorf("coudln't read json file or its content is invalid")
```

Comment:
Maybe we should use the following msg: `coudln't read json file or provided json is invalid`.
Since user will see this message when he provided invalid json string